src/modules/Builder/features/ActivityItems/ItemConfiguration/__tests__/ItemConfigurationDisplayedContent.test.tsx

```diff
@@ -44,14 +65,14 @@ describe('ItemConfiguration: Displayed Content', () => {
     ${'[[Item5]]'}                          | ${'Remove the variable referring to the skipped item.'}         | ${'cannot have item variable which is skippable'}
     ${'[[ItemItem]]'}                       | ${'Remove the variable referring to the nonexistent item.'}     | ${'cannot refer to non-existent item'}
   `('$description', async ({ text, expected }) => {
-    const ref = createRef();
-
-    renderWithAppletFormData({
-      children: renderItemConfiguration(),
-      formRef: ref,
-    });
+    const ref = renderItemConfig();
 
-    ref.current.setValue(`${mockedItemName}.question`, text);
+    fireEvent.change(
```

Comment:
could you please use `await userEvent.type`? It simulates closer as a user action.